src/components/HomePage.vue

```diff
@@ -2,15 +2,20 @@
   import { ref } from 'vue';
   import ToDoList from './ToDoList.vue';
   import HeaderComponent from './HeaderComponent.vue';
-  import toDoListMT from './../assets/toDoListMT.svg';
+  import toDoListPlaceHolder from './../assets/toDoListPlaceHolder.svg';
+  import PopUpToDoListConfirmation from './popUpToDoListConfirmation.vue';
+  import { newDate } from '../utils/date';
+
   const toDos = ref([]);
+
   function addNewTodo() {
     const newToDo = {
-      title: 'Title',
-      text: 'tanulj Vue.js-t',
+      title: '',
+      text: '',
       priority: 'High',
-      created_at: '2023-01-30',
-      status: 'Active',
+
+      id: newDate(),
+      icon: true,
```

Comment:
Please rename this properties to be more descriptive instead of id and icon should be createdAt and completed